src/components/Feedbacks/MarketDataUnavailable.tsx

```diff
@@ -0,0 +1,19 @@
+import { Alert } from '@chakra-ui/react'
+import { FaInfoCircle } from 'react-icons/fa'
+import { IconCircle } from 'components/IconCircle'
+import { Text } from 'components/Text'
+
+export const MarketDataUnavailable = () => {
+  return (
+    <Alert status='info' variant='subtle' borderRadius='lg' pl={2}>
```

Comment:
Nitpick: Since this is effectively used in one place and we will always want that `padding: 8px` on the two places it is used, you could move the `<Box>` wrapper here.